Update status badge doc (#11990)
MicrosoftDocs/azure-devops-docs: docs/pipelines/includes/get-status-badge.md

1. In Azure Pipelines, go to the **Pipelines** page to view the list of pipelines. Select the pipeline you created in the previous section.

> 2. On the upper right, click on three dots to expand the dropdown menu.

Reviewer: ```suggestion
2. In the upper-right corner, select the three dots to expand the dropdown menu.
```

> 3. Select **Status badge**

Reviewer: ```suggestion
3. Select **Status badge**.
```
